frontend/src/employee-frontend/components/reports/StartingPlacements.tsx

```diff
@@ -117,106 +116,254 @@ export default React.memo(function StartingPlacements() {
             <Wrapper>
               <Combobox
                 items={yearOptions}
-                selectedItem={filters.year}
-                onChange={(year) => {
-                  if (year !== null) {
-                    setFilters({ ...filters, year })
+                selectedItem={year.value()}
+                onChange={(newYear) => {
+                  if (newYear !== null) {
+                    year.set(newYear)
                   }
                 }}
                 placeholder={i18n.common.year}
               />
             </Wrapper>
           </FlexRow>
         </FilterRow>
+        <StartingPlacements filters={filters.value()} />
+      </ContentArea>
+    </Container>
+  )
+})
 
-        <FilterRow>
-          <FilterLabel>{i18n.reports.common.careAreaName}</FilterLabel>
-          <FlexRow>
-            <Wrapper>
-              <Combobox
-                items={[
-                  { value: '', label: i18n.common.all },
-                  ...rows
-                    .map((rs) =>
-                      distinct(rs.map((row) => row.careAreaName)).map((s) => ({
-                        value: s,
-                        label: s
+type SortColumn = keyof StartingPlacementsRow
+const StartingPlacements = React.memo(function StartingPlacements({
+  filters
+}: {
+  filters: PlacementsReportFilters
+}) {
+  const { i18n } = useTranslation()
+  const result = useQueryResult(startingPlacementsReportQuery(filters))
+  useEffect(() => {
+    setDisplayFilters(emptyDisplayFilters)
+  }, [filters])
+
+  const [displayFilters, setDisplayFilters] =
+    useState<DisplayFilters>(emptyDisplayFilters)
+  const areaFilter = (row: StartingPlacementsRow): boolean =>
+    !(displayFilters.careArea && row.careAreaName !== displayFilters.careArea)
+  const displayFilter = (row: StartingPlacementsRow): boolean =>
+    areaFilter(row) &&
+    !(displayFilters.unit && row.unitName !== displayFilters.unit)
+
+  const [sort, setSort] = useState<{
+    columns: SortColumn[]
+    direction: SortDirection
+  }>({
+    columns: ['careAreaName', 'unitName', 'lastName', 'firstName'],
+    direction: 'ASC'
+  })
+
+  const sortBy = useCallback(
+    (columns: SortColumn[]) => {
+      if (isEqual(sort.columns, columns)) {
+        setSort({
+          columns,
+          direction: sort.direction === 'ASC' ? 'DESC' : 'ASC'
+        })
+      } else {
+        setSort({ columns, direction: 'ASC' })
+      }
+    },
+    [sort.columns, sort.direction]
+  )
+  const sorted = useCallback(
+    (columns: SortColumn[]) =>
+      isEqual(sort.columns, columns) ? sort.direction : undefined,
+    [sort.columns, sort.direction]
+  )
+  return (
+    <>
+      {renderResult(result, (rawResult) => {
+        const rows = orderBy(rawResult.filter(displayFilter), sort.columns, [
+          sort.direction === 'ASC' ? 'asc' : 'desc'
+        ])
```

Comment:
this could be memoized with something like
```
const sortedResult = useMemo(() =>
   result.map(rawResult => orderBy(rawResult.filter(displayFilter), sort.columns, [
      sort.direction === 'ASC' ? 'asc' : 'desc'
   ])),
   |result, displayFilter, sort]
)

return (
   <>
      {renderResult(sortedResult, (rows) => {
```
but maybe it's not too heavy anyway